agent/src/main/java/com/teamscale/jacoco/agent/upload/artifactory/ArtifactoryConfig.java

```diff
@@ -118,6 +127,13 @@ public static boolean handleArtifactoryOptions(ArtifactoryConfig options,
 			case ARTIFACTORY_PASSWORD_OPTION:
 				options.password = value;
 				return true;
+			case ARTIFACTORY_LEGACY_PATH_OPTION:
+				if (value.isEmpty()) {
```

Comment:
I would suggest to get rid of this special handling to be consistent with all other boolean options in the agent i.e. `dump-on-exit`, `ignore-uncovered-classes`, `validate-ssl`, ...
IMO it's less confusing to give the user one clear option instead of two and also it would feel strange to omit the value in the properties file.